README.md

```diff
@@ -0,0 +1,27 @@
+# tennis
+# This project controls tennis game scores and shows feedback on how the score looks currently
+
+## Run game simulation:
+```
+clj -M:run
+```
+## Run tests:
+
+clj -M:test
+
+
+## Run the repl:
+```
+clj
+(load-file "src/tennis/core.clj")
+(use 'tennis.core)
+```
+
+
+Available functions:
+```
+(tennis-set-score :player1) ;; add game to player1 
+(tennis-set-score :player2) ;; add game to player2
+(tennis-set-score :random)  ;; add game to random player
```

Comment:
This is something which I would be careful with. If you extend the task with new commands you have to document it. Now, after going through the code I know that `:random` means that one of the players (a random player) will score the point but at the first look I was surprised why we have a new, fourth command here.

Reply:
I renamed :random to :random-player